R/tsd.R

```diff
@@ -1,24 +1,23 @@
-#' Create a S3 `aedseo_tsd` (time-series data) object from observed data and corresponding dates.
+#' Create a S3 `tsd` (time-series data) object from observed data and corresponding dates.
```

Comment:
```suggestion
#' Create a tibble-like `tsd` (time-series data) object from observed data and corresponding dates.
```

I don't think most users will need to know what S3 is, so I suggest dropping it from the docs.